gammapy/catalog/fermi.py

```diff
@@ -1669,7 +1669,7 @@ def pulse_profile_best_fit(self):
         best_fit: `~gammapy.maps.RegionNDMap`
             Map containing the best fit.
         """
-        table = Table.read(self._auxiliary_filename, hdu=2)
+        table = Table.read(self._auxiliary_filename, hdu="BEST_FIT_LC")
```

Comment:
it table is None, the code is protected?

Reply:
Currently it raises: `KeyError: "Extension 'RADIO_PROFILE' not found."`. Which is kind of explicit but I could improve the error message.